web/home/src/pages/Home/home.tsx

```diff
@@ -0,0 +1,153 @@
+import { useEffect, useState } from 'react';
+
+import massaLogomark from '../../assets/massa_logomark_detailed.png';
+
+import axios from 'axios';
+import { PluginHomePage } from '../../../../shared/interfaces/IPlugin';
+import { PluginCard } from '../../components/pluginCard';
+
+import Header from '../../components/Header';
+
+import ManagePluginCard from '../../components/managePluginCard';
+import grid1 from '../../assets/element/grid1.svg';
+import wallet from '../../assets/logo/plugins/Wallet.svg';
+import registry from '../../assets/logo/plugins/Registry.svg';
+import webOnChain from '../../assets/logo/plugins/WebOnChain.svg';
+import MainTitle from '../../components/MainTitle';
+
+/**
+ * Homepage of Thyra with a list of plugins installed
+ *
+ */
+
+function Home() {
+  // Fetch plugins installed by calling get /plugin/manager
+
+  const fakePluginsList: PluginHomePage[] = [
+    {
+      name: "Massa's Wallet",
+      description:
+        'Create and manage your smart wallets to buy, sell, transfer and exchange tokens',
+      id: '420',
+      logo: wallet,
+      status: 'Up',
+      home: '/thyra/wallet'
+    },
+    {
+      name: 'Web On Chain',
+      description:
+        'Buy your .massa domain and upload websites on the blockchain',
+      id: '421',
+      logo: webOnChain,
+      status: 'Up',
+      home: '/thyra/websiteCreator'
+    },
+    {
+      name: 'Registry',
+      description: 'Browse Massa blockchain and its .massa websites',
+      id: '423',
+      logo: registry,
+      status: 'Up',
+      home: '/thyra/registry'
+    },
+  ];
+  const [plugins, setPlugins] = useState<PluginHomePage[]>(fakePluginsList);
+
+  const handleOpenPlugin = (pluginName: string) => {
+    window.open(findPluginHome(pluginName), '_blank');
+  };
+  const findPluginHome = (pluginName: string) => {
+    let home = '';
+    plugins.forEach((element) => {
+      if (element.name == pluginName && element.home) {
+        return element.home;
+      }
+    });
+    console.log('Link to the plugin not found')
+    return home;
+  };
+  // List of plugins
+  const getPlugins = async () => {
+    const init = {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+    const res = await axios.get(`/plugin-manager`, init);
+    return res.data;
+  };
```

Comment:
I think our headers would be always with **json** content type, isn't it!? 🤔 I am not sure using axios but I think it will work by default. It  wouldn't work if you remove the header conf?

In any case, I would place this request in a specific file for REST actions. Specially if we need reuse that in future. Specially if we need to check for errors, wrapping that in a try/catch, etc... it can be hard to scale if we keep it here in the same place.

Reply:
Yes clearly , i think we can share the header to all request and regroup this into axios.ts as it's done in Plugin Manager page. 

At the rebranding we plan to merge Homepage and PluginManagerPage in one project and we will use the axios.ts from plugin Manager Page. So we can accept it will be solve at the merge of the two projects 😃